common/src/main/java/io/homeassistant/companion/android/common/data/keychain/KeyChainRepository.kt

```diff
@@ -0,0 +1,17 @@
+package io.homeassistant.companion.android.common.data.keychain
+
+import android.content.Context
+import java.security.PrivateKey
+import java.security.cert.X509Certificate
+
+interface KeyChainRepository {
+    suspend fun getAlias(): String?
+
+    suspend fun load(context: Context, alias: String): Boolean
+
+    suspend fun load(context: Context): Boolean
```

Comment:
The `Boolean` return types of these methods are not used anywhere, as I saw. I think we could remove them.

Reply:
Done